server/src/main/java/org/elasticsearch/search/aggregations/metrics/TDigestExecutionHint.java

```diff
@@ -21,6 +22,14 @@ public enum TDigestExecutionHint implements Writeable {
     DEFAULT(0),        // Use a TDigest that is optimized for performance, with a small penalty in accuracy.
     HIGH_ACCURACY(1);  // Use a TDigest that is optimize for accuracy, at the expense of performance.
 
+    public static final Setting<String> SETTING = Setting.simpleString(
```

Comment:
I think `Setting#enumSetting(...)` should be used here?

Reply:
That one is case sensitive - setting to HIGH_ACCURACY works but high_accuracy doesn't. That's why I think using ::parse is preferable here?

Reply:
Right, I see this method always uses `Enum.valueOf(...)` which is case sensitive and only allows values as is defined in the enum itself. (and in `ExecutionHint` all enums are defined in uppercase).

Ok, let's keep it this way.